client/components/payment-data-highlight/index.tsx

```diff
@@ -0,0 +1,99 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import ArrowUpIcon from 'gridicons/dist/arrow-up';
+import ArrowDownIcon from 'gridicons/dist/arrow-down';
+import { __ } from '@wordpress/i18n';
+import { VisuallyHidden } from '@wordpress/components';
+import { Link } from '@woocommerce/components';
+import { recordEvent } from 'tracks';
+
+/**
+ * Internal dependencies
+ */
+import { ClickTooltip } from 'components/tooltip';
+import './styles.scss';
+
+const PaymentChangeFlow: React.FunctionComponent< {
+	change: number;
+} > = ( { change } ): JSX.Element => {
```

Comment:
Since both cases look pretty similar (only some minor class name changes), I wonder if we can just set some variables representing the class names which need to change based on if `change < 0` to avoid having to write the markup twice. 🙂

Reply:
Thank you, @dmallory42! Indeed, that block could be enhanced to avoid duplicate markup. Improved with f9f641b.